commands/state.bee.inc

```diff
@@ -0,0 +1,291 @@
+    return;
+  }
+}
+
+/**
+ * Command callback: Set the value of a Backdrop state.
+ */
+function state_set_bee_callback($arguments, $options) {
+  try {
+    $state_name = (string) $arguments['state'];
+    $value = $arguments['value'];
+    // Check the type of the existing value
+    $existing_value = state_get($state_name);
+    $existing_type = gettype($existing_value);
+    switch ($existing_type) {
+      case 'integer':
+        // Make sure input is a number and then ensure its past as integer
+        if (is_numeric($value)) {
+          $value = (int) $value;
+        }
+        else {
+          // Reject input that is invalid
+          $err_msg = bt("'!state' is an !type state and the argument you entered is not a valid !type.", array(
+            '!type' => $existing_type,
+            '!state' => $state_name,
+          ));
+          bee_message((string) $err_msg, 'error');
+          return;
+        }
+        break;
+      case 'boolean':
+        // Check input is valid boolean - allow true/false and 1/0
+        $value = strtoupper($value);
+        switch ($value) {
+          case 'TRUE':
+            $value = (bool) TRUE;
+            break;
+          case '1':
+            $value = (bool) TRUE;
+            break;
+          case 'FALSE':
+            $value = (bool) FALSE;
+            break;
+          case '0':
+            $value = (bool) FALSE;
+            break;
+          default:
+            // Reject invalid input
+            $err_msg = bt("'!state' is a !type state and the argument you entered is not a valid !type. Enter 'TRUE' or 'FALSE' (not case sensitive) OR '1' or '0'.", array(
+              '!type' => $existing_type,
+              '!state' => $state_name,
+            ));
+            bee_message((string) $err_msg, 'error');
+            return;
+          }
+        break;
+      case 'string':
+        // Pass string as is but cast to string to be safe
+        $value = (string) $value;
+        break;
+      case 'array':
+        // Do not allow updating of array states
+        $err_msg = bt("'!state' is an !type and cannot be set via the command line.", array(
+          '!type' => $existing_type,
+          '!state' => $state_name,
+        ));
+        bee_message((string) $err_msg, 'error');
+        return;
+      case 'NULL':
+        // If type is NULL then state does not exist and we do not want to
+        // allow user to set it.
+        $err_msg = bt("State '!state' cannot be found. New states cannot be created from the command line. Please check your spelling and try again.", array(
+          '!state' => $arguments['state'],
+        ));
+        bee_message((string) $err_msg, 'error');
+        return;
+      default:
+        // Reject any other types - no others known at present
+        $err_msg = bt("'!state' is a !type and is not currently enabled to be set via the command line. Please submit an issue listing the state, the type, and what module has created it.", array(
```

Comment:
```suggestion
        $err_msg = bt("'The type of the !state' state is !type, which is not currently supported to be set via the command line. Please submit an issue listing the state, the type, and what module has created it.", array(
```

Reply:
You can't download `bee` with the project browser so not sure anyone would get bee that didn't know about github.